doc/rpc/pathfinder_rpc_api.json

```diff
@@ -295,6 +322,44 @@
                     "ABORTED"
                 ],
                 "description": "The status of a transaction"
+            },
+            "WebsocketEventNewHead": {
+                "type": "object",
+                "properties": {
+                    "block_hash": {
+                        "type": "string"
+                    },
+                    "block_number": {
+                        "type": "number"
+                    },
+                    "gas_price": {
+                        "type": "string"
+                    },
+                    "parent_block_hash": {
+                        "type": "string"
+                    },
+                    "sequencer_address": {
+                        "type": "string"
+                    },
+                    "state_commitment": {
+                        "type": "string"
+                    },
+                    "status": {
+                        "type": "string",
+                        "enum": [
+                            "PENDING",
+                            "ACCEPTED_ON_L2",
+                            "ACCEPTED_ON_L1",
+                            "REJECTED"
+                        ]
+                    },
+                    "timestamp": {
+                        "type": "number"
+                    },
+                    "starknet_version": {
+                        "type": "string"
+                    }
+                }
```

Comment:
It would be good to mark these as `required`, an [example](https://github.com/starkware-libs/starknet-specs/blob/master/api/starknet_api_openrpc.json#L1219-L1262) from the starknet specs.